getting_hired/preparing_to_interview.md

```diff
@@ -63,7 +63,7 @@ You'll need to read up on a variety of things that weren't focused on in the pre
 * [Project Euler](http://projecteuler.net/) has more generic and challenging problems that must be solved efficiently (they can be very computationally intensive).
 * [Code Wars](https://www.codewars.com) has programming problems and examples of best practice. Join 'The Odin Project' clan for allies.
 * [Coding Bat](http://codingbat.com/) has practice questions in Java and Python.
-* [HackerRank](https://www.hackerrank.com/) provides challenges, drills, and competitions on algorithms and data scructures. 
+* * [HackerRank](https://www.hackerrank.com/) provides challenges, drills, and competitions on algorithms and data structures.
```

Comment:
Hi @sanket-r-jain, it looks like you have an extra `*` at the start of this line. Can you remove it please?

Reply:
Hi Kevin, thanks for pointing out the error. I have fixed it now & have created a new pull request which can be found here - https://github.com/TheOdinProject/curriculum/pull/22669
Please let me know if we need to make any other changes.